collaboration-service/index.js

```diff
@@ -0,0 +1,103 @@
+import express from "express";
+import cors from "cors";
+import { createServer } from "http";
+import mongoose from "mongoose";
+import "dotenv/config";
+import { Server, Socket } from "socket.io";
+import Room from "./model/room-model.js";
+import collaborationRoutes from "./routes/collaboration-routes.js";
+
+const uri =
+  process.env.ENV == "PROD"
+    ? process.env.DB_CLOUD_URI
+    : process.env.DB_LOCAL_URI;
+
+mongoose
+  .connect(uri)
+  .then((x) =>
+    console.log(
+      `Connected to MongoDB! Database name: "${x.connections[0].name}"`
+    )
+  )
+  .catch((err) => console.error("Error connecting to MongoDB", err.reason));
+
+const app = express();
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+app.use(cors());
+app.options("*", cors());
```

Comment:
I think can remove this line as `app.use(cors())` is sufficient

https://stackoverflow.com/questions/52942286/comparing-implementing-app-usecorscorsoptions-and-app-options-cors

Reply:
Not 100% sure on this. Actually just leave it as I think it may be needed unless you're sure it can be removed. 

https://stackoverflow.com/questions/58914404/socket-io-cors-error-by-using-node-react-and-socket-io